src/RestControllers/FhirEncounterRestController.php

```diff
@@ -12,28 +12,28 @@
 
 namespace OpenEMR\RestControllers;
 
-use OpenEMR\Services\EncounterService;
+use OpenEMR\Services\FhirEncounterService;
```

Comment:
services moved so
use OpenEMR\Services\FHIR\FhirEncounterService;

check for all your other controllers. i'm lazy to point them out

Reply:
Done!